girder_annotation/girder_large_image_annotation/web_client/views/hierarchyWidget.js

```diff
@@ -0,0 +1,105 @@
+import $ from 'jquery';
+
+import { wrap } from '@girder/core/utilities/PluginUtils';
+import { restRequest } from '@girder/core/rest';
+import AccessWidget from '@girder/core/views/widgets/AccessWidget';
+import HierarchyWidget from '@girder/core/views/widgets/HierarchyWidget';
+
+import AnnotationModel from '../models/AnnotationModel';
+
+wrap(HierarchyWidget, 'initialize', function (initialize, settings) {
```

Comment:
Since we are doing the same thing in the render method, can we get rid of doing it in the initialize method?

Reply:
Yes, we can.

See 62a6b50